docs/content/en/docs/observability/loki.md

```diff
@@ -39,52 +39,111 @@ For this case, we'll use Helm.
 - kubectl and kubeconfig configured
 
 
-#### Install Loki and Promtail
+#### Install Grafana and Loki
 
 From a terminal, run the following commands to add the Loki repo and then install Loki
 
 ```bash
-$ helm repo add loki https://grafana.github.io/loki/charts
+$ helm repo add grafana https://grafana.github.io/helm-charts
 $ helm repo update
-$ helm upgrade --install loki loki/loki-stack
+$ helm upgrade -n monitoring --create-namespace --install loki-grafana grafana/loki-stack \
+--set grafana.enabled="true" \
+--set promtail.enabled="false"
 ```
 
-This will install Loki in the default namespace.
-Check if there're pods running for Loki and Promtail.
+This will install Loki & Grafana in the monitoring namespace.
+Check if there're pods running for Loki and Grafana.
+
 
-#### Install Grafana
+#### Install Promtail
 
-Similarly, to install Grafana, run the following commands from a terminal.
+You'll notice that the Promtail installation is disabled above. This is because custom configuraiton
```

Comment:
should this information be put before loki-stack installation where the promtail flag is disabled?

Reply:
That wouldn't work in this case since Promtail requires lokiAddress to be set. Else Promtail will log a bunch of connection errors on startup.

I think the better solution might to be install loki, promtail, grafana separately rather than via the loki-stack chart.

Reply:
hey @kbbqiu , I was referring to the text `You'll notice that the Promtail installation is disabled above...... ` if this should be mentioned before the command that has `--set promtail.enabled="false"` - so that the reader has clear understanding.